### PR TITLE
Feature/mime dictionary

### PR DESCRIPTION
**Pull request by klaseskilson**
Use a MimeType dictionary for both pretty file types and future file view template choosing.

### CODE REVIEW

File: app/assets/javascripts/factories/MimeDictionary.coffee [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    'application/vnd.oasis.opendocument.graphics': 'opendocument graphics'
+    'application/vnd.ms-excel': 'excel spreadsheet'
+    'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet': 'excel spreadsheet'
+    'application/vnd.ms-powerpoint': 'powerpoint presentation'
+    'application/vnd.openxmlformats-officedocument.presentationml.presentation': 'powerpoint presentation'
+    'application/vnd.ms-word': 'word document'
+    'application/vnd.openxmlformats-officedocument.wordprocessingml.document': 'word document'
+    'application/x-iwork-keynote-sffkey': 'keynote presentation'
+    'application/vnd.google-apps.document': 'google drive document'
+    'application/vnd.google-apps.presentation': 'google drive presentation'
```
Comment (ronjagrosz) [inline app/assets/javascripts/factories/MimeDictionary.coffee:75]: These are the default apps in google drive that we probably should support:
'application/vnd.google-apps.map': 'google drive map'
'application/vnd.google-apps.drawing': 'google drive drawing'
'application/vnd.google-apps.form': 'google drive form'
'application/vnd.google-apps.spreadsheet': 'google drive spreadsheet'
'application/vnd.google-apps.fusiontable': 'google drive fusiontable'

**Comment by ronjagrosz** [comment]
I think it looks good. The dictionary can be expanded when needed, but this is a very good start!

**Comment by tkomstadius** [comment]
I also think it looks good, should be easy to expand. I think it is ready for merge, perhaps the google mimes should be added first though?

**Comment by klaseskilson** [comment]
@tkomstadius, @Fluffpannkaka. The dictionary has been updated with the google drive mime types.

**Comment by tkomstadius** [comment]
najs, looks pretty

**Comment by ronjagrosz** [comment]
Merging!

**Pull request merged**